### PR TITLE
[mds-db] [mds-policy] [mds-policy-author] [mds-compliance] #82 Add indexed identity column to all tables, #108 Implement `mds-policy-author` API, #102 Update Policy type

### PR DESCRIPTION
**Pull request by avatarneil**
Mostly pay attention to the last two merges by @avatarneil, the first merge was heavily scrutinized.

Go through commit by commit, much easier to parse.

## Impacts
- [x] Policy
- [x] Compliance

### CODE REVIEW

**Review by mdurling — DISMISSED** [review]

**Review by mdurling — DISMISSED** [review]
Re-approving after merge latest origin/master

**Review by janedotx — DISMISSED** [review]

**Review by janedotx — DISMISSED** [review]
looks legit

File: packages/mds-policy/tests/api.spec.ts
Comment (janedotx) [inline packages/mds-policy/tests/api.spec.ts]: Shouldn't there be a case where unpublished policies are viewable?

**Review by janedotx — DISMISSED** [review]

**Review by janedotx — APPROVED** [review]

**Pull request merged**
